_data/pt-BR.yml

```diff
@@ -0,0 +1,294 @@
+  hTradeDuration: "Quanto tempo leva uma negociação?"
+  pTradeDuration: >
+    <p>A duração de cada negociação é determinada pelos tempos de transferência das moedas que você esta usando. Moedas nacionais podem ser instantâneas (Faster Payment) ou até levar alguns dias (Transferência bancária).</p> <p>Transações em Bitcoin levam cerca de 10 minutos para confirmar e é preciso ter pelo menos uma confirmação para que o comprador envie seu fiat ou altcoins.</p> <p>O período de negociação de Altcoins é sempre de 1 dia (ou 1 hora para Alcoins com Negociação instantânea).</p><p>Por favor consulte a <a href="https://docs.bisq.network/payment-methods" target="_blank">lista de meios de pagamento</a> para todos os detalhes.</p>
+  hTradeCost: "Quanto custa a negociação na Bisq?"
+  pTradeCost: >
+    <p>Para negociar com a Bisq, os usuários pagam (1) taxas de negociação para a Bisq e (2) taxas de mineração para os mineradores.</p>
+
+    <strong>Taxas de negociação</strong> <p>As taxas de negociação são pagas em BTC ou BSQ. BSQ é o bitcoin colorido que a Bisq usa para o seu próprio financiamento e governança. O seu uso é opcional, mas existem <a href="https://docs.bisq.network/getting-started-dao-traders.html" target="_blank">benefícios significativos</a>.</p> <div id="faq-tables"> <p class="note">Taxas de negociação de BTC, por 1 BTC negociado:</p> <table> <thead> <tr> <th></th> <th> Taxas em BTC</th> <th>Taxa como % da transação no valor de 1 BTC</th> </tr> </thead> <tbody> <tr> <td>Taxa do ofertante</td> <td>0.002 BTC</td> <td>0.20%</td> </tr> <tr> <td>Taxa do tomador</td> <td>0.006 BTC</td> <td>0.60%</td> </tr> </tbody> </table> <p class="note">Taxas de negociação de BSQ, por 1 BTC negociado:</p> <table> <thead> <tr> <th></th> <th>Taxas em BSQ</th> <th>Taxa como % da transação no valor de 1 BTC</th> </tr> </thead> <tbody> <tr> <td>Taxa do ofertante</td> <td>13 BSQ</td> <td>0.13%</td> </tr> <tr> <td>Taxa do tomador</td> <td>38 BSQ</td> <td>0.38%</td> </tr> </tbody> </table> </div> <p>As porcentagens de taxa em BSQ acima assumem uma taxa de 0.00001 BSQ/BTC.</p> <p>As taxas nominais de BTC e BSQ são ajustadas periodicamente pela votação da DAO para acomodar as flutuações do mercado.</p> <p>As taxas mínimas de negociação são fixadas em 0,00005 BTC e 0,03 BSQ para evitar limites com poeira.</p>
```

Comment:
This FAQ isn't updated with current fees.

See:
https://bisq.network/faq/#cost-to-trade